### PR TITLE
Er 286 postcode anywhere and legal entity implementation

### PR DESCRIPTION
**Pull request by sdholder**

### CODE REVIEW

File: src/Employer/Employer.Web/Startup.Configure.cs [hunk 1]
```diff
@@ -46,7 +46,8 @@ public void Configure(IApplicationBuilder app, IHostingEnvironment env, IOptions
                     s.Self()
                     .CustomSources("https://az416426.vo.msecnd.net", 
                                     "https://www.google-analytics.com/analytics.js", 
-                                    $"https://www.googletagmanager.com/")
+                                    $"https://www.googletagmanager.com/",
+                                    "http://services.postcodeanywhere.co.uk/")
```
Comment (chrisjensenuk) [inline src/Employer/Employer.Web/Startup.Configure.cs]: Can we use https?

File: src/Employer/Employer.Web/Views/Part1/Employer/Employer.cshtml [hunk 2]
```diff
@@ -67,4 +69,42 @@
...
+        key: '@PcaConfig.Value.Key',
+        findEndpoint: '@PcaConfig.Value.FindEndpoint',
+        retrieveEndpoint: '@PcaConfig.Value.RetrieveEndpoint'
+    };
+</script>
+}
+@section FooterJS
+{
+<script asp-condition="@Model.HasMoreThanOneOrganisation">
+    var addresses = @Json.Serialize(Model.Organisations);
```
Comment (chrisjensenuk) [inline src/Employer/Employer.Web/Views/Part1/Employer/Employer.cshtml]: Should we move `addresses` & `setAddresssById` out of global? Maybe move these into the `$(document).ready` function?

File: src/Employer/Employer.Web/Views/Shared/_Layout.cshtml [hunk 3]
```diff
@@ -74,9 +74,12 @@ public string GetTitle()
     <environment exclude="@EnvironmentNames.GetNonProdEnvironmentNamesCommaDelimited()">
     @Html.Partial(PartialNames.GoogleAnalyticsSnippet)
     </environment>
-    @Html.Raw(JavaScriptSnippet.FullScript) <!-- AppInsights script -->
+    @Html.Raw(JavaScriptSnippet.FullScript) <!-- AppInsights script -->
+    <script>
+        window.EsfaRecruit = {};
```
Comment (chrisjensenuk) [inline src/Employer/Employer.Web/Views/Shared/_Layout.cshtml]: 👍

File: src/Employer/Employer.Web/Views/Part1/Employer/Employer.cshtml (hunk 2 above)
Comment (sachpatel) [reply src/Employer/Employer.Web/Views/Part1/Employer/Employer.cshtml]: 👍

File: src/Employer/Employer.Web/Startup.Configure.cs (hunk 1 above)
Comment (sachpatel) [reply src/Employer/Employer.Web/Startup.Configure.cs]: Yes 👍

**Pull request merged**